index.html

```diff
@@ -42,8 +42,168 @@ <h1>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <span class="see-project btn">See Project</span>
+      </div>
+
+      <div class="work-card card-4">
+        <div class="work-card-image-container">
+          <img src="./images/Snapshoot_4.jpg" alt="First work image">
+        </div>
+        <h1>Multi-Post Stories</h1>
+        <div class="work-card-properties-container">
+          <span>CANOPY</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>Back End Dev</span>
+          <span><img src="./images/Counter.jpg" alt="dot"></span>
+          <span>2015</span>
+        </div>
+        <p>
+          A daily selection of privately personalized reads;
+           no accounts or sign-ups required.
+        </p>
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <span class="see-project btn">See Project</span>
+      </div>
+
+    </section>
+
+    <!-- About Me Section -->
+    <section class="about">
+      <h1>About Myself</h1>
+      <p class="about-details">
+        Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <span class="lets-connect">LET'S CONNECT</span>
+      <ul class="social-icons">
+        <li><a href="https://twitter.com/raykingflamez"><i class="fab fa-twitter"></i></a></li>
+        <li> <a href="https://www.linkedin.com/in/obatola-david-96a32378/"><i class="fab fa-linkedin-in"></i></a></li>
+        <li><a href="https://github.com/Raykingflamez"><i class="fab fa-github"></i></a></li>
+      </ul>
+      <div class="get-my-resume btn">
+        <a href="" class="get-my-resume-link">Get my resume</a>
+      </div>
+
+      <div class="stack-details">
+        <ul>
+          <li class="stack-detail">
+            <span class="languages">
+              <p>Languages</p>
+              <i class="fas fa-angle-down"></i>
+            </span>
+            <ul>
+              <li>
+                <img src="./images/js.png" alt="">
+                <span>Javascript</span>
+              </li>
+              <li>
+                <img src="./images/html.png" alt="">
+                <span>HTML</span>
+              </li>
+              <li>
+                <img src="./images/css.png" alt="">
+                <span>CSS</span>
+              </li>
+            </ul>
+          </li>
+
+          <li class="stack-detail">
+            <span class="frameworks">
+              <p>Frameworks</p> 
+              <i class="fas fa-angle-right"></i>
+            </span>
+          </li>
+          
+          <li class="stack-detail">
+            <span class="skills">
+              <p>Skills</p> 
+              <i class="fas fa-angle-right"></i>
+          </li>
+        </ul>
```

Comment:
Thanks for the kind review and corrections but uhm, please give me more clarification on this because that's what I did for the technology categories, only making the first entries in each list a div that has the name of the technologies as well as their icon. Do you mean to say that I should make every entry in the top-level ul be encased within the bottom-level li?
@bobb-Rob